client/actions/multiSelect.js

```diff
@@ -157,7 +157,13 @@ const exportAsArticle = (items = []) => (
             const itemIds = items.map((item) => item.id);
 
             if (download) {
-                window.open(`${getState().config.server.url}/planning_download/events/${itemIds.join(',')}`, '_blank');
+                let queryString = `${getState().config.server.url}/planning_download/events/${itemIds.join(',')}`;
```

Comment:
Use `selectors.config.getServerUrl(getState())` instead of directly accessing the state